ocs_ci/ocs/ui/page_objects/s3_vector_tab.py

```
@@ -0,0 +1,429 @@
+        if metric_lower == "cosine":
+            locator_key = "distance_metric_cosine"
+        elif metric_lower == "euclidean":
+            locator_key = "distance_metric_euclidean"
+        else:
+            raise ValueError(
+                f"Unknown distance metric: '{distance_metric}'. Use 'cosine' or 'euclidean'."
+            )
+        self.do_click(self.s3_vector_loc[locator_key])
+        logger.info(f"Selected distance metric radio button: {distance_metric}")
+
+    def _select_data_type(self, data_type: str):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_select_data_type`

Add return type annotation: `None`

(ANN202)